driver/src/models/account_state.rs

```diff
@@ -1,32 +1,13 @@
-use ethcontract::{Address, H256, U256};
-use serde::{Deserialize, Serialize};
+use ethcontract::Address;
 use std::collections::HashMap;
 
-#[derive(Serialize, Default, Deserialize, Clone, Debug, PartialEq)]
-#[serde(rename_all = "camelCase")]
-pub struct AccountState {
-    pub state_hash: H256,
-    pub state_index: U256,
-    balances: HashMap<Address, HashMap<u16, u128>>, // UserId => (TokenId => balance)
-    pub num_tokens: u16,
-}
+// (UserId TokenId) => balance
```

Comment:
nit: Maybe use a doc comment here that is a little more descriptive. I feel like just specifying the mapping doesn't add much value as that is already described by the type.